sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/README.md

```diff
@@ -1,36 +1,145 @@
 # Azure WebJobs Event Hubs client library for .NET
 
-TODO
+This extension provides functionality for accessing Azure Event Hubs in Azure Functions.
 
 ## Getting started
 
 ### Install the package
 
-Install the Event Grid extension with [NuGet][nuget]:
+Install the Event Hubs extension with [NuGet](https://www.nuget.org/packages/Microsoft.Azure.WebJobs.Extensions.EventHubs):
 
 ```Powershell
 dotnet add package Microsoft.Azure.WebJobs.Extensions.EventHubs --version 5.0.0-beta.1
 ```
 
 ### Prerequisites
 
-TODO
+- **Azure Subscription:**  To use Azure services, including Azure Event Hubs, you'll need a subscription.  If you do not have an existing Azure account, you may sign up for a [free trial](https://azure.microsoft.com/free) or use your [Visual Studio Subscription](https://visualstudio.microsoft.com/subscriptions/) benefits when you [create an account](https://account.windowsazure.com/Home/Index).
+
+- **Event Hubs namespace with an Event Hub:** To interact with Azure Event Hubs, you'll also need to have a namespace and Event Hub available.  If you are not familiar with creating Azure resources, you may wish to follow the step-by-step guide for [creating an Event Hub using the Azure portal](https://docs.microsoft.com/azure/event-hubs/event-hubs-create).  There, you can also find detailed instructions for using the Azure CLI, Azure PowerShell, or Azure Resource Manager (ARM) templates to create an Event Hub.
+
+[![Deploy button](http://azuredeploy.net/deploybutton.png)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-sdk-for-net%2Fmaster%2Fsdk%2Fevent hub%2FAzure.Messaging.EventHubs%2Fassets%2Fsamples-azure-deploy.json)
```

Comment:
You may want to point this as the Processor samples template, as that includes a Storage account and this template doesn't.   If you decide to to so, the only change is appending `.Processor` after `Messaging.EventHubs`.   Otherwise, the pathing and names are the same.